src/elasticodbc/es_info.cpp

```diff
@@ -661,7 +661,7 @@ ESAPI_Tables(HSTMT hstmt, const SQLCHAR *catalog_name_sql,
         //  arguments
         TableResultSet result_type = TableResultSet::All;
 
-        // TODO: Refactor logic
+        // TODO #324 (SQL Plugin)- Refactor logic
```

Comment:
I think this ticket was referring to the logic of the if else statements here, not related to SQL plugin

Reply:
removed